Convert ESRI style to QGIS style xml - unrecognised header value?

**Describe the bug**
Attempting to convert an [ESRI style file](https://esri-styles.maps.arcgis.com/home/item.html?id=c46a93768f064a50814174c48bec10fe) to a QGIS xml style file using the SLYR pluggin. Appears that the header value `Code` is not recognised.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to Processing Toolbox > SLYR > Style databases > Convert ESRI style to QGIS style xml
2. Set the style database to the ESRI style file
3. Set the destination xml file using the save to file option
4. Click run
5. See error below

```
Traceback (most recent call last):
File "C:\Users/***/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\slyr_community\qgis_plugin\algorithms\style_to_xml.py", line 197, in processAlgorithm
raw_symbols = Extractor.extract_styles(input_file, symbol_type)
File "C:\Users/***/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\slyr_community\bintools\extractor.py", line 271, in extract_styles
assert False, header
AssertionError: Code

Execution failed after 0.20 seconds
```

**Desktop:**
 - OS: Win10
 - QGIS 3.24.1
 - SLYR 4.0.1

--- a/slyr_community/bintools/extractor.py
+++ b/slyr_community/bintools/extractor.py
@@ -267,8 +267,6 @@ class Extractor:
                         tags_idx = idx
                     elif header.lower().strip() == 'id':
                         symbol_id_idx = idx
-                    else:
-                        assert False, header
                 continue
 
             res = r.split(Extractor.__DELIMITER)

